src/packagers/npm.ts

```diff
@@ -269,10 +275,13 @@ export class NPM implements Packager {
     return lockfile;
   }
 
-  async install(cwd: string, extraArgs: Array<string>) {
+  async install(cwd: string, extraArgs: Array<string>, useLockfile: boolean) {
     const command = /^win/.test(process.platform) ? 'npm.cmd' : 'npm';
 
-    const args = ['install', ...extraArgs];
+    const args = [
+      !this.packagerOptions.ignoreLockfile && useLockfile ? 'ci' : 'install',
```

Comment:
I see it is indeed looks like breaking change, but I agree with the fix, it is pretty much in alignment with yarn behaviour `ci` has the same behaviour as yarn's `--frozen-lockfile`